tools/licenses/lib/paths.dart

```diff
@@ -0,0 +1,385 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'regexp_debug.dart';
+
+final Set<String> skippedPaths = <String>{
+  // directories or files that are not included in the final binaries
```

Comment:
These must be full paths relative to the repo root, right? If so that's worth mentioning explicitly.

Reply:
added several comments to this file to explain these lists better